code/modules/security levels/security levels.dm

```diff
@@ -21,24 +21,28 @@
 	if(level >= SEC_LEVEL_GREEN && level <= SEC_LEVEL_DELTA && level != security_level)
 		switch(level)
 			if(SEC_LEVEL_GREEN)
+				world << sound('sound/misc/notice2.ogg')
 				to_chat(world, "<font size=4 color='red'>Attention! Security level lowered to green</font>")
```

Comment:
Any way you can do to station z level instead?

Reply:
Yes, maybe in a different PR?

Reply:
bugger, clicked the wrong line, was talking about the world << sound one, but I see what you're saying